src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+                    }
+                }
+                else if (items.get(i) instanceof Event) {
+                        Event dummy = (Event) items.get(i);
+                        System.out.println("[" + items.get(i).returnType() + "] " + "[" + items.get(i).getStatusIcon() + "] " + items.get(i).getDescription() + " (" + dummy.getDuration() + ")");
+                        try {
+                            writeToFile(file1, "[" + items.get(i).returnType() + "] " + "[" + items.get(i).getStatusIcon() + "] " + items.get(i).getDescription() + " (" + dummy.getDuration() + ")");
+                        } catch (IOException e) {
+                            System.out.println("Something went wrong: " + e.getMessage());
+                        }
+                } else {
+                    Todo dummy = (Todo) items.get(i);
+
+                    System.out.println("[" + items.get(i).returnType() + "] " + "[" + items.get(i).getStatusIcon() + "] " + items.get(i).getDescription());
+                    try {
+                        writeToFile(file1, "[" + items.get(i).returnType() + "] " + "[" + items.get(i).getStatusIcon() + "] " + items.get(i).getDescription());
+                    } catch (IOException e) {
+                        System.out.println("Something went wrong: " + e.getMessage());
+                    }
+
+                }
+                }
+
+                System.out.println("\n____________________________________________________________");
+            }
+        }
+
+
+
+    public static void doneTodo(ArrayList<Todo> items, String order){
+        String dummy = order.trim();
+        int index = Integer.parseInt(dummy.substring(5,dummy.length()));
+        items.get(index-1).markAsDone();
+        System.out.println("\n____________________________________________________________");
+        System.out.println("Nice Work! I've marked it as done");
+
+        System.out.println("\n____________________________________________________________");
+    }
+
+    public static void greeting () {
+
+        ArrayList<Todo> items = new ArrayList<>();
+        int maxcount = 0;
+        boolean flag = true;
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n");
+
+        while(flag) {
+            try {
+                String order = getMission();
+
+
+                while (!order.equals("bye")) {
+                    if (order.equals("list")) {
+                        showList(items, maxcount);
+                        order = getMission();
+                        continue;
+                    } else if (order.contains("done")) {
+                        doneTodo(items, order);
+                        order = getMission();
+                        continue;
+                    }
+                    if (order.contains("todo")) {
+                        items.add(new Todo(order.substring(5, order.length())));
```

Comment:
Consider extracting out these printing statements into an external method for clarity (SLAP hard),